setup.py

```diff
@@ -56,8 +44,6 @@
     long_description_content_type="text/markdown",
     classifiers=[
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
```

Comment:
We should add a matrix to the testing above to cover platforms and versions and then expand this up through 3.11 accordingly.  Well, happy to have that be a separate PR instead of this one.  Just pointing it out.

There should also be a `python_requires=`.

https://github.com/Chia-Network/chia-blockchain/blob/6af4f86171b40554f12126bdfd3c1a548e5fa1b6/setup.py#L84
```python
    python_requires=">=3.8.1, <4",
```